lib/ansible/module_utils/gce.py

```diff
@@ -27,9 +27,13 @@
 # USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #
 
+import json
 import os
 import traceback
 
+from distutils.version import LooseVersion
+
+import libcloud
```

Comment:
this should be try/except wrapped to avoid import exceptions on modules, see ec2.py for examples